src/index.js

```diff
@@ -132,8 +136,8 @@ class Client extends EventEmitter {
 			return;
 		}
 
-		if (bloks.login_response && bloks.login_response.logged_in_user.pk) {
-			this.userId = bloks.login_response.logged_in_user.pk_id;
+		if (bloks.login_response.logged_in_user.pk) {
+			this.userId = bloks.login_response.logged_in_user.pk;
```

Comment:
`pk_id` is better because it's a string and we don't use numbers

Reply:
Oh yes, of course. That means we will just stick to using pk_id throughout.